CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # cuSpatial 0.14.0 (Date TBD)
 
 ## New Features
-
+- PR 143 construct quadtree on large-scale point data
```

Comment:
Should be a new line break after this. Also, surprised Github isn't complaining about merge conflict as this file is clearly out-of-date.